robotiq_3f_gripper_articulated_gazebo_plugins/package.xml

```diff
@@ -27,6 +27,7 @@
 
   <build_depend>angles</build_depend>
   <build_depend>message_generation</build_depend>
+  <build_depend>gazebo</build_depend>
```

Comment:
Should this be `gazebo_ros` instead?

Reply:
no, gazebo_ros is already a listed. Catkin_lint gives the following error:

```
robotiq_3f_gripper_articulated_gazebo_plugins: error: missing build_depend on 'gazebo'
catkin_lint: checked 17 packages and found 1 problems
```